packages/emails/email-manager.ts

```diff
@@ -22,10 +25,12 @@ import OrganizerRequestReminderEmail from "./templates/organizer-request-reminde
 import OrganizerRequestedToRescheduleEmail from "./templates/organizer-requested-to-reschedule-email";
 import OrganizerRescheduledEmail from "./templates/organizer-rescheduled-email";
 import OrganizerScheduledEmail from "./templates/organizer-scheduled-email";
-import TeamInviteEmail, { TeamInvite } from "./templates/team-invite-email";
+import type { TeamInvite } from "./templates/team-invite-email";
+import TeamInviteEmail from "./templates/team-invite-email";
 
 export const sendScheduledEmails = async (calEvent: CalendarEvent) => {
   const emailsToSend: Promise<unknown>[] = [];
+  const clonedEvent = cloneDeep(calEvent);
```

Comment:
Since we need to alter the attendee emails to hide attendees if needed. Let's clone the event to pass to the organizer emails